src/components/BlogRoll_en.js

```diff
@@ -6,11 +6,8 @@ import { getCurrentLangKey, getLangs, getUrlForLang } from 'ptz-i18n';
 class BlogRoll extends React.Component {
 
   render() {
-    const { data } = this.props
-    const { edges: posts } = data.allMarkdownRemark
-    const url = location.pathname;
```

Comment:
The code above was causing the issue, and was'not neede at all, probably code  that was trying time ago...